exercises/practice/parallel-letter-frequency/.docs/instructions.md

```diff
@@ -0,0 +1,7 @@
+# Instructions
+
+Count the frequency of letters in texts using parallel computation.
+
+Parallelism is about doing things in parallel that can also be done sequentially.
+A common example is counting the frequency of letters.
+Create a function that returns the total frequency of each letter in a list of texts and that employs parallelism.
```

Comment:
I know this comes in from problem specification but that does not mean that we have to live with it here in this track.  There are tracks that do not have functions, tracks that do not have methods, tracks that do not have lists, etc.

Since this language does not have functions, it seems strange to request that a function is created.

Just a point of contention, not saying that we must do anything about it.

Reply:
Can you open an issue for this in Problem Specs please, @kotp. It should be fixed there really.